src/components/Company/Offers/Manage/CompanyOffersManagementWidget.js

```diff
@@ -1,14 +1,18 @@
-import { Typography } from "@material-ui/core";
-// import { format, parseISO } from "date-fns";
+import { IconButton, TableCell } from "@material-ui/core";
+import { format, parseISO } from "date-fns";
 import React, { useState, useEffect } from "react";
 import { fetchCompanyOffers } from "../../../../services/companyOffersService";
 import ControlledSortableSelectableTable from "../../../../utils/Table/ControlledSortableSelectableTable";
 import FilterableTable from "../../../../utils/Table/FilterableTable";
-import { generateTableCellFromField } from "../../../../utils/Table/utils";
+import { alphabeticalSorter, generateTableCellFromField } from "../../../../utils/Table/utils";
 import { columns } from "./CompanyOffersManagementSchema";
 import PropTypes from "prop-types";
-import { format, parseISO } from "date-fns";
 import useSession from "../../../../hooks/useSession";
+import { OfferTitleFilter, PublishDateFilter, PublishEndDateFilter, LocationFilter } from "../Filters/index";
+import { Edit as EditIcon } from "@material-ui/icons";
+import { Link } from "react-router-dom";
+// import { RowActions } from "../Actions";
+
 
 const CompanyOffersNonFullfilledRequest = ({ isLoading, error }) => {
     if (isLoading) {
```

Comment:
We should change the loading message to a skeleton (similar to the offer page), or something similar.

Reply:
I'm not familiar with this actually, could you elaborate what loading message you mean?

Reply:
@monkin77 
Instead of displaying this when the table is loading:
![Screenshot from 2021-09-20 10-12-48](https://user-images.githubusercontent.com/50210213/133980162-a94d2943-707b-4171-893e-3724e7409221.png)

you can display a skeleton similar to this (the actions icons should also be a skeleton, my bad):
![Untitled](https://user-images.githubusercontent.com/50210213/133982152-d3b6b130-8ec3-4cb5-bded-41215e72075c.png)

For that, you can use the [Material-UI Skeleton](https://material-ui.com/components/skeleton/) for each grey rectangle in the image. The number of rows of the loading table should be always the same (for example 3, 4, or 5). Notice that this way the user does not need to wait for the request to be completed to see the table, which improves the UX and makes the loading seem much faster.

You can start by removing the `isLoading` prop and condition from the `<CompanyOffersNonFulfilledRequest>` and add that prop to the `<FilterableTable>`. Then, you can do something similar to the `emptyMessage` case, but with the skeleton rows:
![Screenshot from 2021-09-20 10-51-47](https://user-images.githubusercontent.com/50210213/133984017-9df6392d-fa42-4f31-930a-79003c778ad0.png)

Maybe we can do the same for the Review Applications table, as the base table is the same.

Reply:
Also, I think you can remove the `<CompanyOffersNonFulfilledRequest>` component and add a snackbar, and maybe show the `No applications here` if there is some error.

Reply:
I've tried to replicate this behavior. If there's anything missing hit me up